site/content/overview/gateway-api-compatibility.md

```diff
@@ -20,7 +20,7 @@ docs: "DOCS-1412"
 | [TCPRoute](#tcproute)                 | Not supported       | Not supported          | Not supported                         | N/A         |
 | [UDPRoute](#udproute)                 | Not supported       | Not supported          | Not supported                         | N/A         |
 | [BackendTLSPolicy](#backendtlspolicy) | Supported           | Supported              | Not supported                         | v1alpha3    |
-| [Custom policies](#custom-policies)   | Not supported       | N/A                    | Not supported                         | N/A         |
+| [Custom policies](#custom-policies)   | N/A                 | N/A                    | Supported                             | N/A         |
 {{< /bootstrap-table >}}
```

Comment:
Since you're already updating this file, would you mind updating:

- GatewayClass table to have implementation-specific support
- Gateway table to have "Partially Supported" for Extended support
- GRPCRoute to have "Partially Supported" for Extended support